app/views/prevalidateTrader/PrevalidateTraderStartPage.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import views.html.components._
+@import views.html.templates.Layout
+
+@this(
+        layout: Layout,
+        h1: h1,
+        h2: h2,
+        p: p,
+        link: link
+)
+@()(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = titleNoForm(messages("prevalidateTrader.start.title")), maybeShowActiveTrader = maybeShowActiveTrader(request), maybeShowNavigationBanner = None) {
+
+    @h2("prevalidateTrader.caption", classes = "govuk-caption-l hmrc-caption-l", hiddenContent = Some("base.captionHiddenText"))
+    @h1(classes = "govuk-heading-l") {
+        @messages("prevalidateTrader.start.h1")
+    }
+
+    @p()(Html(messages("prevalidateTrader.start.p1")))
+
+    @link(
+        link = testOnly.controllers.routes.UnderConstructionController.onPageLoad().url,
```

Comment:
Is it worth a TODO here? Do we know what this link is going to be?

Reply:
Good spot